stdlib/test/builtin/test_tuple.mojo

```diff
@@ -56,11 +56,57 @@ def test_tuple_contains():
     var d = (123, True, String("Mojo is awesome"))
 
     assert_true(String("Mojo is awesome") in d)
+    assert_false("Mojo is awesome" in d)
     assert_true(d.__contains__(String("Mojo is awesome")))
 
     assert_false(String("Hello world") in d)
     assert_false(d.__contains__(String("Hello world")))
 
+    alias a_alias = (123, True, "Mojo is awesome")
+
+    assert_true("Mojo is awesome" in a_alias)
+    assert_true(a_alias.__contains__("Mojo is awesome"))
+
+    assert_false("Hello world" in a_alias)
+    assert_false(a_alias.__contains__("Hello world"))
+
+    assert_true(123 in a_alias)
+    assert_true(a_alias.__contains__(123))
+
+    assert_true(True in a_alias)
+    assert_true(a_alias.__contains__(True))
+
+    assert_false(False in a_alias)
+    assert_false(a_alias.__contains__(False))
+
+    assert_false(a_alias.__contains__(1))
+    assert_false(a_alias.__contains__(0))
+    assert_false(1 in a_alias)
+    assert_false(0 in a_alias)
+
+    alias b_alias = (False, True)
+    assert_true(True in b_alias)
+    assert_true(b_alias.__contains__(True))
+    assert_true(False in b_alias)
+    assert_true(b_alias.__contains__(False))
+    assert_false(b_alias.__contains__(1))
+    assert_false(b_alias.__contains__(0))
+
+    alias c_alias = (1, 0)
+    assert_false(c_alias.__contains__(True))
+    assert_false(c_alias.__contains__(False))
+    assert_false(True in c_alias)
+    assert_false(False in c_alias)
+
+    alias d_alias = (123, True, String("Mojo is awesome"))
+
+    assert_true(String("Mojo is awesome") in d_alias)
+    assert_false("Mojo is awesome" in d_alias)
```

Comment:
This will surprise users until `StringLiteral` materializes to a `String`.  When this happens @laszlokindrat, would this sort of gotcha go away, or no still since this is comp-time domain?

Reply:
This is not compile time domain, so this will go away when `StringLiteral` becomes NM. Also, eventually we will have the language features so that we can have types that are comparable to other types parametrically.

Reply:
Oh fair point, `assert_false` is already runtime domain despite `d_alias` being comp-time.

Reply:
It should be possible to add more tests, asserting the aliases with:
```mojo
@parameter
if "ok" in a_alias: ...
else: raise("assertion failed")
```
Should i create them ?

Reply:
I wouldn't bother right now; I just landed this guy internally.  WDYT @laszlokindrat @abduld?